FreshAir/UI/RZFReleaseNotesView.m

```diff
@@ -27,35 +30,57 @@
 static const CGFloat kRZFDoneButtonTopPadding = 0.0f;
 static const CGFloat kRZFDoneButtonHeight = 64.0f;
 
+static const CGFloat kRZFDoneButtonFullScreenHorizontalPadding = 6.0f;
+static const CGFloat kRZFDoneButtonFullScreenBottomPadding = 6.0f;
+static const CGFloat kRZFDoneButtonFullScreenHeight = 54.0f;
+static const CGFloat kRZFDoneButtonFullScreenTopPadding = 6.0f;
+
 @interface RZFReleaseNotesView () <UICollectionViewDelegate, UICollectionViewDataSource>
 
 @property (strong, nonatomic) UICollectionView *featureCollectionView;
 @property (strong, nonatomic) UIPageControl *featurePageControl;
 @property (strong, nonatomic) UIButton *doneButton;
 @property (strong, nonatomic) NSArray<RZFFeatureViewModel *> *features;
```

Comment:
Looks like the view is already grabbing the view model, so never mind my previous comment unless you want.